src/Header/Header.tsx

```diff
@@ -15,23 +15,23 @@ interface PropTypes {
 const useStyles = makeStyles(theme => ({
   tab: {
     '& .MuiTab-wrapper': {
-        padding: theme.spacing(2),
-        flexDirection: 'row',
-        fontSize: '0.8125rem'
+      padding: theme.spacing(2),
+      flexDirection: 'row',
+      fontSize: '0.8125rem'
     }
   }
 }));
 
-const tabs = ["Profile", "Feed"];
+const tabs = ['Profile', 'Feed'];
 
-const Header: React.FC<PropTypes> = ({ page, setPage }) => {
+const Header: React.FC<PropTypes> = ({ page /* , setPage */ }) => {
```

Comment:
It is my mistake, I actually forgot to implement `hangeChange` function :)
I will fix it in this PR